spec/features/planting_reminder_spec.rb

```diff
@@ -23,25 +23,16 @@ def self.default_url_options
     scenario "doesn't list plantings" do
       expect(mail).not_to have_content "most recent plantings you've told us about"
     end
-
   end
 
   context "when member has some plantings" do
-    before :each do
-      @p1 = FactoryGirl.create(:planting,
-        :garden => member.gardens.first,
-        :owner => member
-      )
-      @p2 = FactoryGirl.create(:planting,
-        :garden => member.gardens.first,
-        :owner => member
-      )
-    end
+    let(:p1) { create :planting, garden: member.gardens.first, owner: member }
+    let(:p2) { create :planting, garden: member.gardens.first, owner: member }
 
     scenario "lists plantings" do
       expect(mail).to have_content "most recent plantings you've told us about"
```

Comment:
So I think what's happening here is

- we refer to `mail`, which is created lazily. Since there are no plantings at this point, the mail contains no information about them.
 - we refer to `p1` and `p2`, which are created lazily. There are now plantings to talk about, but it's too late! The mail has already been created without them.

So we somehow have to force the instantiation of `p1` and `p2` before `mail`. This is kinda ugly: I think the test needs a comment explaining the order-dependence.